kolibri/plugins/user/templates/user/user.html

```diff
@@ -1,8 +1,10 @@
 {% extends "kolibri/base.html" %}
 {% load user_tags %}
 {% load kolibri_tags %}
+{% load webpack_tags %}
 {% load cache %}
 {% block content %}
   {% user_async_assets %}
   {% user_assets %}
+  {% webpack_asset 'user_module' %}
```

Comment:
I am making this change in a parallel PR so there'll be a merge conflict here, happy to resolve any of the plugin related merge conflicts that arise!